src/testing/mocks.rs

```diff
@@ -62,8 +62,7 @@ impl<'a, L: reef::Ledger> MockStorage<'a, L> {
     }
 }
 
-#[async_trait]
-impl<'a, L: reef::Ledger> KeystoreStorage<'a, L> for MockStorage<'a, L> {
+impl<'a, L: reef::Ledger> MockStorage<'a, L> {
```

Comment:
Can't we delete the whole `MockStorage` type now?

Reply:
Yep! All gone now